src/AppBundle/Serializer/ContributorNormalizer.php

```diff
@@ -67,6 +70,9 @@ public function normalize($object, $format = null, array $context = []): array
             'ratings' => [
                 'subscribes' => $object->getActiveSubscriptionsCount(),
             ],
+            'notices' => $object->getPublicNotices()->map(function (Notice $notice) {
```

Comment:
`noticesUrls` to make it clear we don’t have subobjects here …

Reply:
Agreed. Since you mention this subject, I was confused by the field `contributions` which is the number of contributions and not objects :roll_eyes:

Reply:
I agree :-/